examples/indexer/src/indexer.ts

```diff
@@ -69,15 +74,37 @@ export function createIndexerConfig(streamUrl: string) {
       "run:before"() {
```

Comment:
```suggestion
      async "run:before"() {
```

The hook handlers can be async

Reply:
Then adjust the code in the handle to use `await`.